apps/meteor/app/lib/server/lib/passwordPolicy.ts

```diff
@@ -0,0 +1,162 @@
+import { PasswordPolicy } from '@rocket.chat/password-policies';
+
+import { settings } from '../../../settings/server';
+
+let enabled = false;
+let minLength = -1;
+let maxLength = -1;
+let forbidRepeatingCharacters = false;
+let forbidRepeatingCharactersCount = 3;
+let mustContainAtLeastOneLowercase = false;
+let mustContainAtLeastOneUppercase = false;
+let mustContainAtLeastOneNumber = false;
+let mustContainAtLeastOneSpecialCharacter = false;
+
+export let passwordPolicy = new PasswordPolicy({
+	enabled,
+	minLength,
+	maxLength,
+	forbidRepeatingCharacters,
+	forbidRepeatingCharactersCount,
+	mustContainAtLeastOneLowercase,
+	mustContainAtLeastOneUppercase,
+	mustContainAtLeastOneNumber,
+	mustContainAtLeastOneSpecialCharacter,
+	throwError: true,
+});
+
+settings.watch('Accounts_Password_Policy_Enabled', (value) => {
+	enabled = Boolean(value);
+	passwordPolicy = new PasswordPolicy({
```

Comment:
Maybe it would be good to create a function to create this instance instead of repeating for all the settings 😬

Reply:
Correct! I improved the code